### DISCUSSION TITLE
Global stack_tags group config

### DISCUSSION THREAD

**Discussion by zaro0508** [cloudreach/sceptre#732] [Q&A]
We use sceptre and have many individual sceptre stack configs to deploy our templates.  We use the `sceptre launch prod --yes` command to deploy all our stacks at once.  I would like to apply the same tag to all of the stacks that are deployed or updated.  Is there a way to do this without having to define [stack_tags](https://sceptre.cloudreach.com/latest/docs/stack_config.html#stack_tags) in every sceptre stack config file?

**Comment by mike-rsi** [comment]
This is the best way I've found to do it so far: https://github.com/cloudreach/sceptre/issues/256#issuecomment-499954300
